keps/0036-kudo-operator-toggle-behavior.md

```diff
@@ -0,0 +1,147 @@
+---
+kep-number: 36
+short-desc: KudoOperator Toggling
+title: Parameter-controlled toggling of KudoOperator tasks
+authors:
+  - "@asekretenko"
+owners:
+  - "@asekretenko"
+editor: @asekretenko
+creation-date: 2021-04-30
+last-updated: 2021-04-30
+status: provisional
+see-also:
+  - KEP-23
+  - KEP-29
+---
+
+# Parameter-controlled toggling of KudoOperator tasks
+
+## Table of Contents
+* [Summary](#summary)
+* [Motivation](#motivation)
+  * [Goals](#goals)
+  * [Non-Goals](#non-goals)
+* [Proposal](#proposal)
+  * [Interface](#interface)
+     * [Validation](#validation)
+     * [Changing enablingParameter on upgrade](#changing-enablingparameter-on-upgrade)
+  * [Implementation Notes](#implementation-notes)
+  * [Risks and Mitigations](#risks-and-mitigations)
+* [Drawbacks](#drawbacks)
+* [Alternatives](#alternatives)
+
+<!-- [Tools for generating]: https://github.com/ekalinin/github-markdown-toc -->
+
+## Summary
+
+This KEP aims to add an ability to switch an operator dependency on and off
+via a specified parameter of a parent operator. Namely, as a result of
+implementing this KEP, `KudoOperator` task definition will get an optional field
+containing a name of a boolean parameter (of the parent operator), the value
+of which will determine whether the child operator instance should exist.
```

Comment:
this doesn't make sense...  in particular `should exist`.   I'm guessing.. but perhaps you mean that the operator will expect that the child operator exists and will not be created as part of a deployment.   

Also... it might be worth editing this paragraph with a focus on "why" and less on how.  The how will be outlined in the KEP.. the summary should strongly support "why" it adds value.  what the world is likely without and what the world will be like with it.... which then be amplified by the motivation below.